zhaquirks/philips/__init__.py

```diff
@@ -168,25 +167,30 @@ def handle_cluster_request(
     ):
         """Handle the cluster command."""
         _LOGGER.debug(
-            "PhilipsRemoteCluster - handle_cluster_request tsn: [%s] command id: %s - args: [%s]",
+            "%s - handle_cluster_request tsn: [%s] command id: %s - args: [%s]",
+            self.__class__.__name__,
```

Comment:
Not sure if you are fine with this. Main purpose is to show `PhilipsRdmRemoteCluster` instead of `PhilipsRemoteCluster` in the logs for RDM002 devices. Can remove it if you want. Same a couple lines down.

Reply:
Hmm, I guess it should be fine.

Can you also use `self.debug` instead of `_LOGGER.debug` and remove the logger? 
That should include the NWK address of the actual device in the logs.

Reply:
To get back to this again: I've just thought about this the other day and while including the Nwk address of the device is nice, this also changes the logging prefix to look like it's coming from zigpy (and not zhaquirks).

This will (hopefully) be generally addressed in the future, as quirks sometimes use the zigpy mixin and sometimes a new logger at the moment.

For now, please use a custom logger (again). (Sorry for the back and forth with this one)